tpm2/tpm2.go

```diff
@@ -461,40 +477,53 @@ func ReadPublic(rw io.ReadWriter, handle tpmutil.Handle) (Public, []byte, []byte
 	return decodeReadPublic(resp)
 }
 
-func decodeCreate(in []byte) ([]byte, []byte, error) {
-	var resp struct {
-		Handle       tpmutil.Handle
-		Private      []byte
-		Public       []byte
-		CreationData []byte
-	}
+type createResponse struct {
```

Comment:
Remove this and use a long list of returns from `decodeCreate`, like `decodeCreatePrimary`